packages/components/src/icons/tokens/crvBUSDIcon.tsx

```diff
@@ -0,0 +1,4882 @@
+import * as React from 'react';
+import SvgIcon from '@material-ui/core/SvgIcon';
+
+import { TokenIconProps } from '../types';
+
+export function crvBUSDIcon(props: TokenIconProps) {
```

Comment:
Use SvgIconProps instead of TokenIconProps, because TokenIconProps contains inactive prop